pontoon/sync/tasks.py

```diff
@@ -250,6 +266,13 @@ def sync_project_repo(self, project_pk, repo_pk, project_sync_log_pk, now, obsol
                 if not no_commit and locale in changeset.locales_to_commit:
                     commit_changes(db_project, vcs_project, changeset, locale)
 
+                log.info(
+                    'Synced locale {locale} for project {project}.'.format(
+                        locale=locale.code,
+                        project=db_project.slug,
+                    )
+                )
+
```

Comment:
Adding a few more log messages shouldn't hurt. If syncing a project with many locales, we only printed log messages if there was something to commit before the final `Synced translations for project {0} in locales {1}` log message.